.ruby-version

```diff
@@ -0,0 +1 @@
+2.7.1
```

Comment:
I don't think so. 2.7.1 is a minimum version. Because I used filter_map.

Reply:
I understand. Just added it due to GHA config but removed it now to use multiple versions of Ruby in CI.